model-optimizer/extensions/middle/RandomUniformReplacer.py

```diff
@@ -64,3 +65,80 @@ def replace_pattern(graph: Graph, match: Dict[str, Node]):
                                                      op_attrs={'name': node_name + '/Broadcast', 'mode': 'numpy'})
         node.in_port(0).get_connection().set_destination(broadcast_node.in_port(1))
         node.out_port(0).get_connection().set_source(broadcast_node.out_port(0))
+
+
+class DropoutReplacer(MiddleReplacementPattern):
+    """
+    Dropout (for inference mode) containing RandomUniform can be replaced with Broadcast of ones in sub-graph.
+
+    Dropout block:
+    ShapeOf -> RandomUniform -> Mul ---> Add ---> Add -> Floor
+                                /\       /\       /\
+                                |        |        |
+                            Const(1)   Const(1)  Const(0)
+
+    Resulted block:
+    ShapeOf -> Broadcast
+                   /\
+                   |
+               Const(1)
+    """
+    enabled = True
+
+    def run_before(self):
+        return [RandomUniformReplacer]
+
+    @staticmethod
+    def pattern():
+        return dict(
+            nodes=[
+                ('shape', dict(op='ShapeOf')),
+                ('shape_data', dict()),
+                ('random_uniform', dict(op='RandomUniform')),
+                ('random_uniform_data', dict()),
+                ('mul', dict(op='Mul')),
+                ('mul_data', dict()),
+                ('mul_const', dict(op='Const')),
+                ('mul_const_data', dict(value=1)),
```

Comment:
fix constants for the transformation by analogy from `extensions/front/HSwish_fusion.py`